app/src/main/java/com/fstyle/androidtrainning/screen/main/MainViewPagerAdapter.java

```diff
@@ -3,33 +3,39 @@
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentPagerAdapter;
-import com.fstyle.androidtrainning.screen.main.fragments.another.AnotherFragment;
-import com.fstyle.androidtrainning.screen.main.fragments.mysong.MySongFragment;
-import com.fstyle.androidtrainning.screen.main.fragments.online.OnlineFragment;
+import com.fstyle.androidtrainning.screen.main.mainfragments.another.AnotherFragment;
+import com.fstyle.androidtrainning.screen.main.mainfragments.mysong.MySongFragment;
+import com.fstyle.androidtrainning.screen.main.mainfragments.online.OnlineFragment;
+import com.fstyle.androidtrainning.utils.Constant;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.utils.Constant. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports